java-grupa8/personal project/.gitignore

```diff
@@ -0,0 +1,2 @@
+target
```

Comment:
Your binaries (.class) are still in the commit.
Check the path - they are located under /out and not /target. Please update the gitignore file.